base/applications/notepad/main.c

```diff
@@ -27,6 +27,10 @@
 #include <shlobj.h>
 #include <strsafe.h>
 
+#if defined(_MSC_VER) && !defined(NDEBUG)
```

Comment:
NDEBUG is mostly for "debug output" . I think you need to use `_DEBUG` instead?

Reply:
Fixed.